gradle/any/dependencies.gradle

```diff
@@ -202,8 +202,14 @@ libraries["http-builder-ng-okhttp"] = "io.github.http-builder-ng:http-builder-ng
 
 ////////////////////////////// TDS ///////////////////////////////////////////
 
-libraries["aws-java-sdk-s3"] = "software.amazon.awssdk:s3:2.10.7"
-//libraries["aws-java-sdk-s3"] = dependencies.create("com.amazonaws:aws-java-sdk-s3:1.11.236") {
+
+libraries["awssdk-s3"] = "software.amazon.awssdk:s3:2.10.7"
+libraries["awssdk-sdk-core"] = "software.amazon.awssdk:sdk-core:2.16.42"
+
+libraries["aws-sdk-core"] = "com.amazonaws:aws-java-sdk-core:1.11.999"
+libraries["aws-sdk-s3"] = "com.amazonaws:aws-java-sdk-s3:1.11.999"
+
```

Comment:
All four of these deps. are required b/c the code imports all of those namspaces at compile time.

Reply:
just hit the link to the file, I didn't choose that snippet of code, Github did it for me

Reply:
I think the versions could prolly be trued-up...